keyboards/draculad/readme.md

```diff
@@ -0,0 +1,14 @@
+# dracuLad
```

Comment:
Would you mind getting this file a bit more in line with our [template](https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md)?